README.md

```diff
@@ -202,6 +204,17 @@ Phantoms will no longer spawn if the hostile mobcap is full. This is per player.
 - Required options: `true`, `false`
 - Categories: `FEATURE`, `CARPET-ADDONS-NOT-FOUND`
 
+### portalCreativeDelay
```

Comment:
Should say `portalSpectatorDelay`

Reply:
this is what i get for copying the description from base carpet at midnight